src/components/BContainer/BContainer.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <section>
+    <component :is=headerElement class="header" v-if="label">{{ label }}</component>
+    <div>
+      <slot></slot>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  name: 'b-container',
+  props: {
+    label: {
+      type: String    
+    },
+    level: {
+      type: [String, Number],
+      default: 1,
+      validator: function(value) {
+        value = Number(value)
+        return !!value && (value >= 0 && value <= 6)
```

Comment:
Could we throw an error here so it won't render? With only returning a boolean it will still mount invalid elements and only give a Vue warning.

Reply:
Sure.